src/main/java/redis/clients/jedis/BuilderFactory.java

```diff
@@ -1341,6 +1342,63 @@ public AggregationResult build(Object data) {
     }
   };
 
+  public static final Builder<ExhgetWithVerResult<String>> EXHGETWITHVER_RESULT_STRING
+      = new Builder<ExhgetWithVerResult<String>>() {
+    @Override
+    @SuppressWarnings("unchecked")
+    public ExhgetWithVerResult<String> build(Object data) {
+      if (data == null) {
+        return null;
+      }
+      List<Object> result = (List<Object>)data;
+      if (result.size() == 0) {
+        return null;
+      } else {
+        String value = SafeEncoder.encode((byte[])result.get(0));
```

Comment:
check result size

Reply:
As long as the return value is not empty, it must be two elements, and the engine guarantees, so there is no need to do check